README.md

```diff
@@ -128,8 +128,11 @@ Follow the given instructions for Login into the app.
 	6. Click Create & View.
 	7. The API KEY is generated and displayed to you just once. So be sure to copy and save it somewhere.
 
+3. `SECRET_KEY`  - Add your secret key in the .env file. 
+
 Add it to your .env file as follows:
 ```
+export SECRET_KEY=<your-secret-key>
```

Comment:
Actually, If we use dotenv then there is no need of using the export keyword in the file. We can directly define the environment variables.
Or If we don't want to add it in the .env file then we can simply export it.
I hope it makes clear to you.
So Please change the third point and make it more descriptive.